arrow-libs/fx/arrow-fx-coroutines/src/commonMain/kotlin/arrow/fx/coroutines/Bracket.kt

```diff
@@ -20,6 +20,14 @@ public sealed class ExitCase {
   }
 }
 
+@PublishedApi
+internal val ExitCase.errorOrNull: Throwable?
```

Comment:
Should we just make this `public`?

Reply:
I think the edge-case of needing this outside of the library implementation would be infrequent enough (and this method is trivial enough) to not warrant it.

Reply:
Agreed. Will leave as-is for now